examples/platform/silabs/SiWx917/SiWx917/rsi_if.c

```diff
@@ -48,6 +53,25 @@
 #define VAL(str) #str
 #define TOSTRING(str) VAL(str)
 
+#ifdef CHIP_DEVICE_CONFIG_ENABLE_SED
+
+void M4_sleep_wakeup();
+#ifdef RSI_M4_INTERFACE
+
+#define FLASH_BASED_EXECUTION_ENABLE 1
+#define WKP_RAM_USAGE_LOCATION 0x24061000 /*<!Bootloader RAM usage location upon wake up  */
+
+#ifdef COMMON_FLASH_EN
+#define IVT_OFFSET_ADDR 0x8212000 /*<!Application IVT location VTOR offset>        */
+#else
+#define IVT_OFFSET_ADDR 0x8012000 /*<!Application IVT location VTOR offset>        */
+#endif
+#define WKP_RAM_USAGE_LOCATION 0x24061000 /*<!Bootloader RAM usage location upon wake up  */
```

Comment:
Where did you get these values? It seems odd that we don't already have defines somewhere for these values.